src/libImaging/JpegEncode.c

```diff
@@ -342,7 +346,7 @@ ImagingJpegEncode(Imaging im, ImagingCodecState state, UINT8 *buf, int bytes) {
             }
             jpeg_finish_compress(&context->cinfo);
 
-cleanup:
+        cleanup:
```

Comment:
If we wanted to not add indentation, we could add in `IndentGotoLabels: false` - https://github.com/hugovk/Pillow/pull/118